packages/ui/src/upper-bar-manager.tsx

```diff
@@ -9,6 +9,23 @@ import {PresetsUtils} from './presets-utils';
 import UpperBarConfig = KalturaPlayerContribTypes.UpperBarConfig;
 import {getContribConfig} from './contrib-utils';
 import {IconsMenu} from './components/icons-menu';
+
+const DefaultPluginOrder: {[pluginName: string]: number} = {
+  Navigation: 10,
+  'Q&A': 20,
+  Moderation: 30,
+  Transcript: 40,
+  'Download transcript': 50,
+  Share: 70,
+  Playlist: 80,
+  Info: 90,
+  Related: 100,
+  Quiz: 110,
+  Layout: 120,
+  'Source Selector': 140,
+  Polls: 150,
```

Comment:
Why some with brackets and some w/o - lets keep all same

Reply:
prettier demands

Reply:
Ok